test/k8s_tests/mocks/base.go

```diff
@@ -0,0 +1,53 @@
+package mocks
+
+import (
+	rcsv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	knativev1 "knative.dev/serving/pkg/apis/serving/v1"
+)
+
+var (
+	NsName = "capp-e2e-tests"
+)
+
+func CreateBaseCapp() *rcsv1alpha1.Capp {
+	return &rcsv1alpha1.Capp{
+		TypeMeta: metav1.TypeMeta{
+			Kind:       "Capp",
+			APIVersion: "rcs.dana.io/v1alpha1",
+		},
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "capp-default-test",
+			Namespace: NsName,
+		},
+		Spec: rcsv1alpha1.CappSpec{
+			ConfigurationSpec: knativev1.ConfigurationSpec{
+				Template: knativev1.RevisionTemplateSpec{
+					ObjectMeta: metav1.ObjectMeta{
+						CreationTimestamp: metav1.Time{},
+					},
+					Spec: knativev1.RevisionSpec{
+						PodSpec: corev1.PodSpec{
+							Containers: []corev1.Container{
+								{
+									Env: []corev1.EnvVar{
+										{
+											Name:  "APP_NAME",
+											Value: "capp-default-test",
+										},
+									},
+									Image:     "sahar2339/example-python-app:v1-flask",
```

Comment:
Change image to use `quay.io/danateamorg/example-python-app:v1-flask` or alternatively upload it to `ghcr.io` in this repository and use that instead